buildspec.yml

```diff
@@ -0,0 +1,63 @@
+version: 0.2
+
+env:
+  variables:
+    FRAMEWORK_FULL_VERSION: '1.4.1'
+    FRAMEWORK_SHORT_VERSION: '1.4'
+    AWS_DEFAULT_REGION: 'us-west-2'
+    ECR_REPO: 'sagemaker-mxnet-serving'
```

Comment:
the PR builds use a test repo: https://github.com/aws/sagemaker-mxnet-container/blob/master/buildspec.yml#L10

Reply:
Using sagemaker-test repo now.